src/deploy.sh

```diff
@@ -187,10 +187,10 @@ function deploy_main()
 
   if [[ "$modules" == 0 ]]; then
     start=$(date +%s)
-    # Update name: release + alias
-    name=$(make kernelrelease)
+    # Update kernel name: release + alias
+    kernel_name=$(get_kernel_release "$flag")
 
-    run_kernel_install "$reboot" "$name" "$flag" "$target" '' "$build_and_deploy"
+    run_kernel_install "$reboot" "$kernel_name" "$flag" "$target" '' "$build_and_deploy"
```

Comment:
Here and below I simply rename it so that the same variable names mean the same thing.

Reply:
Make the change of unifying the names on a separate commit before adding the new functionality.